notes/NewSiteArchitecture.md

```diff
@@ -0,0 +1,174 @@
+# New GraphQL.org Site Architecture:
+
+## Index
+
+*Goal:* This is the landing page and is our opportunity to quickly capture attention and explain what GraphQL is and why you should care.
+
+*Timeframe:* Launch Mon, Sept 12th
+
+This page is effectively a marketing page for GraphQL and should be the visual, scrollable version of the "Introducing GraphQL" conference talks and should be rich with visual metaphor and illustration and take advantage of whitespace to make individual salient points.
+
+Above the fold, this page should succinctly explain what GraphQL is and illustrate with a simple (editable) query/response example. Before scrolling you should understand the following:
+
+* GraphQL is an query language for APIs.
+* GraphQL is sent by client applications, such as an iOS app.
+* GraphQL is evaluated by a web service and often returned as JSON.
+* GraphQL typically replaces REST.
+
+Below the fold we should introduce concepts one at a time, each with visual metaphor and take-aways:
+
+1) GraphQL describes what a client needs in terms of how client developers think about data.
+
+* If you're familiar with JSON, GraphQL is easy to learn and understand.
+* It's easy to anticipate the shape of the result for any query.
+* GraphQL only sends what you ask for, nothing more or less.
+
+2) GraphQL queries can access many "resources" in a single network request.
+
+* A query can access properties of not just one object, but of many related objects.
```

Comment:
Unfortunately, it's not "built-in" in any of the open source server implementations we found - you need to write a custom network layer on top of the graphql execution engine to get batching, which is something you could just as easily do with REST.

I agree that it would be awesome if this were a benefit of GraphQL but it's not quite there yet as an out-of-the-box feature.